codechefcli/decorators.py

```diff
@@ -33,3 +33,32 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
 
     return wrapper
+
+
+def sort_it(func):
```

Comment:
You can use the helper method `sort_data_rows` you created in this one. Don't duplicate the code.

Reply:
pehle waise kara tha na sir...to wo hatana bhul gya...yhi pe rhaega...waha se hata de rha.